### ISSUE TITLE
bigdataviewer-core 10.4.0 breaks compilation

### ISSUE THREAD

**Issue by ctrueden** [fiji/MaMuT#30]
```
$ mvn -Dbigdataviewer-core.version=10.4.2 clean package
... build fails with:
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR :
[INFO] -------------------------------------------------------------
[ERROR] /Volumes/Code/fiji/MaMuT/src/main/java/fiji/plugin/mamut/viewer/MamutViewerPanel.java:[80,35] display has private access in bdv.viewer.ViewerPanel
[ERROR] /Volumes/Code/fiji/MaMuT/src/main/java/fiji/plugin/mamut/viewer/MamutViewerPanel.java:[81,35] display has private access in bdv.viewer.ViewerPanel
[ERROR] /Volumes/Code/fiji/MaMuT/src/main/java/fiji/plugin/mamut/viewer/MamutViewerPanel.java:[86,17] currentAnimator has private access in bdv.viewer.ViewerPanel
[ERROR] /Volumes/Code/fiji/MaMuT/src/main/java/fiji/plugin/mamut/viewer/MamutViewerPanel.java:[87,17] currentAnimator has private access in bdv.viewer.ViewerPanel
[INFO] 4 errors
```

Because bigdataviewer/bigdataviewer-core@25303365c8c7f4f2b87a1307d7330d7e30229ac6 changed some fields from protected to private.

This is a problem for the SciJava BOM because we want to update bigdataviewer-core to version 10.4+, but MaMuT 0.7.5 does not work with those BDV versions.

The solution is to update MaMuT not to use those protected fields anymore.

CC @tinevez @tpietzsch

**Comment by ctrueden** [comment]
It looks like @tpietzsch already fixed very similar problems in bigwarp: https://github.com/saalfeldlab/bigwarp/pull/145

So hopefully the fixes here are analogous.

**Comment by tpietzsch** [comment]
fixed by #31
